client/src/utils/noti.ts

```diff
@@ -22,7 +22,9 @@ export const registerForPushNotificationsAsync = async (): Promise<string | unde
       return;
     }
 
-    token = (await Notifications.getExpoPushTokenAsync()).data;
+    if (Platform.OS !== 'web') {
+      token = (await Notifications.getExpoPushTokenAsync()).data;
+    }
```

Comment:
Well, but we still need to achieve push notification from the web browser in order to send push messages.

@0916dhkim Do you also have any idea about this changes?

Reply:
It seems like there is no way around it. Let's merge this and find out if we can implement notification on web without `expo-notifications`.